src/main.rs

```diff
@@ -48,6 +57,17 @@ async fn main() -> Result<()> {
     let starknet_provider = Arc::new(starknet_provider);
     let eth_provider = EthDataProvider::new(db.clone(), starknet_provider.clone()).await?;
 
+    let validator = KakarotTransactionValidatorBuilder::new(Arc::new(ChainSpec {
+        chain: 0x6b6b_7274.into(),
```

Comment:
I think we should fetch chain id from the provider for example

Reply:
Yep just put that as hardcoded for now but yes we should modify this